src/main/resources/openapi/transcriptions.yaml

```diff
@@ -387,3 +447,51 @@ components:
         description:
           type: string
           example: Standard
+
+    TranscriptionResponse:
+      type: object
+      properties:
+        case_id:
+          type: integer
+          example: 1
+        case_number:
+          type: string
+          example: Swansea_case_1
+        courthouse:
+          type: string
+          example: Swansea
+        defendants:
+          type: array
+          items:
+            $ref: '#/components/schemas/defendant'
+        judges:
+          type: array
+          items:
+            $ref: '#/components/schemas/judge'
+        transcript_file_name:
+          type: string
+          example: file001.docx
+        hearing_date:
+          type: string
+          format: date
+          example: 2023-06-23
+        urgency:
+          type: string
+          example: Standard
+        request_type:
+          type: string
+          example: Specified Times
+        transcription_start_ts:
+          type: string
+          format: time
```

Comment:
The 2 timestamp fields should be `format: date-time`, then in the code you can make use of the `OffsetDateTime` field as seen in the generated model file.  So Jackson serialisation will be handled for you in the response to the ISO DATE TIME string format.

Reply:
Changed timestamp fields to date-time format